qiskit_experiments/curve_analysis/__init__.py

```diff
@@ -498,8 +503,8 @@ def _create_analysis_results(self, fit_data, quality, **metadata):
     GaussianAnalysis
     ErrorAmplificationAnalysis
 
-Fit Functions
-*************
+Fit Functions (Deprecated)
```

Comment:
Why deprecate these? You could still use them to initialize a model as `lmfit.Model(fit_func)` and that might even be clearer than using an ExpressionModel which has to parse a string

Reply:
Removed deprecation. See my comments below.